lib/apiservers/engine/backends/image.go

```diff
@@ -234,6 +234,8 @@ func imageConfigToDockerImageInspect(imageConfig *metadata.ImageConfig, productN
 	return inspectData
 }
 
+const DefaultDockerRegistry = "registry-1.docker.io/"
```

Comment:
This is the default host for the v2 registry that we use, defined in https://github.com/docker/docker/blob/master/registry/config.go#L56. We also define it in imagec.

I've removed the declaration in `image.go` and reused the one from `image_cache.go`.